panels/templates/attractions/events.html

```diff
@@ -134,13 +134,18 @@
       });
     };
 
+    $signupModal.find('.modal-footer .btn.modal-form-switch').on('click', function (event) {
+      $signupModal.modal('hide');
+      $earlySignupModal.modal('show');
+    });
+
     $signupModal.find('.confirm-row .btn-success').on('click', function (event) {
       event.preventDefault();
       var eventId = $signupModal.data('eventId'),
           $badgeNum = $signupModal.find('input[name=badge_num]'),
           badgeNum = $badgeNum.val();
 
-      signupForEvent(eventId, badgeNum, function(response) {
+      signupForEvent(eventId, {badge_num: badgeNum}, function(response) {
```

Comment:
This is cool, but from a quick Google search it looks like it only works in the latest Firefox. That's a bit too bleeding-edge -- unless I'm chasing the wrong lead and this isn't a destructuring assignment.

Reply:
Ah, I see why you'd think that, but that's not what's happening here.

This is actually just passing an object literal as a parameter to this argument.  For example, instead of saying

```javascript
var d = {foo: 'bar'};
someFunc('baz', d, 'baf');
```

We can just say

```javascript
someFunc('baz', {foo: 'bar'}, 'baf'});
```

I can see why it looked like I was doing destructuring assignment, since what I did here looks a lot like the "Setting a function parameter's default value" section at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

However, in this case I'm only calling the function.  If I did something like this while *declaring* a function then you're right that it would only work in bleeding edge browsers and we should (sadly) avoid that kind of thing.

Reply:
Ahh, yes that makes sense. It turns out googling "Javascript curly braces" isn't guaranteed to give context-aware results. Cool! Carry on!